cde-pentaho5/src/pt/webdetails/cdf/dd/api/RenderApi.java

```diff
@@ -405,7 +405,7 @@ public String edit(
     if ( !CdeEnvironment.canCreateContent() ) {
       return "This functionality is limited to users with permission 'Create Content'";
     }
-    return getEditor( path, debug, request.getScheme(), isDefault, response, false );
+    return getEditor( path, debug, request.getScheme(), isDefault, response, true );
```

Comment:
this flag is ultimately used [here](https://github.com/webdetails/cde/blob/master/cde-core/src/pt/webdetails/cdf/dd/editor/DashboardEditor.java#L154) and so when the editor page gets rendered, it asks either for **amd** or **legacy** components, changed it to ask for **amd** components